notification/notification_test.go

```diff
@@ -89,8 +89,8 @@ func TestHandlerSend(t *testing.T) {
 	)
 
 	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		if r.URL.Path != alertPushEndpoint {
-			t.Fatalf("Bad endpoint %q used, expected %q", r.URL.Path, alertPushEndpoint)
+		if r.URL.Path != AlertPushEndpoint {
```

Comment:
As the full URL is now provided externally, this is no longer a behavior of the notification package.
We can remove that check.